aws/lambda/README.md

```diff
@@ -0,0 +1,41 @@
+# AWS Lambda
+
```

Comment:
Thank you for the detailed README :) 

Is it okay to reveal SSO start URL / Account ID / etc in README?
I don't know if these info would be sensitive information but just double checking 👍

Reply:
It's ok, because this configuration process authenticates using a AWS web page that pops up. As long as we do not reveal the login in README or the code, we are safe.